picard/formats/id3.py

```diff
@@ -330,6 +333,20 @@ def _load(self, filename):
                 if frame.desc:
                     name += ':%s' % frame.desc
                 metadata.add(name, frame.text)
+            elif frameid == 'SYLT':
+                name = 'syncedlyrics'
+                if frame.desc:
+                    name += ':%s' % frame.desc
+
+                lrc_lyrics = []
+                for lyrics, milliseconds in frame.text:
+                    minutes = milliseconds // (60 * 1000)
+                    seconds = (milliseconds % (60 * 1000)) // 1000
+                    hundredths = (milliseconds % 1000) // 10
+                    lrc_lyrics.append(f"[{minutes:02d}:{seconds:02d}.{hundredths:02d}]{lyrics.strip()}")
+                lrc_lyrics = "\n".join(lrc_lyrics)
```

Comment:
This is NOT my understanding of the SYLT tags - which are (supposedly) syllable based and have embedded "\n" to indicate a new line. These embedded newlines may well be preserved in addition to those inserted by the  later join.

The LRC format seems to me to be a subset whereby each line of an LRC corresponds to a line of the lyrics, and the lyrics have a single start time-stamp for each line.

This PR therefore needs to have a different approach to newlines than the one here if it is to preserve the SYLT tag exactly as-is. We either need to e.g. simplify the SYLT by converting to LRC and combining multiple syllables under a single time-stamp - in which case data will be lost when you save it - or use a modified LRC format whereby we allow multiple timestamps per line.